README.md

```diff
@@ -163,14 +252,31 @@ Pipes are being used to validate and transform incoming values. The pipes can be
 @Query('isActive', ParseBooleanPipe) isActive: boolean
 ```
 
-⚠️ Beware that they throw when the value is invalid.
+⚠️ Beware that pipes throw when the value is `undefined` or invalid. Read about optional values [here](#handling-optional-values-in-conjunction-with-pipes)
+
+|                    | Description                                       | Remarks                                                                           |
+| ------------------ | ------------------------------------------------- | --------------------------------------------------------------------------------- |
+| `ParseBooleanPipe` | Validates and transforms `Boolean` strings.       | Allows `'true'` and `'false'` as valid values.                                    |
+| `ParseDatePipe`    | Validates and transforms `Date` strings.          | Allows valid `ISO 8601` formatted date strings.                                   |
+| `ParseNumberPipe`  | Validates and transforms `Number` strings.        | Uses `parseFloat` under the hood.                                                 |
+| `ValidateEnumPipe` | Validates string based on `Enum` values.          | Allows strings that are present in the given enum.                                |
+| `ValidationPipe`   | Validates the request body via `class-validator`. | Works only when `class-validator` and `class-transformer` packages are installed. |
+| `DefaultValuePipe` | Assigns a default value to the parameter when its value is `null` or `undefined`. | Bare function usage has no effect. In other words, always use  `@Query('step', DefaultValuePipe(1))` rather than `@Query('step', DefaultValuePipe)`. |
```

Comment:
I think `Bare function usage has no effect.` can be removed and the line can be simplified to `always use `@Query('step', DefaultValuePipe(1))` rather than `@Query('step', DefaultValuePipe)`.`

Reply:
I think keeping it makes a bit more clear. Otherwise people can still assume that bare function style can be used and complain that it's not working.